src/components/dialog-full-screen/dialog-full-screen.scss

```diff
@@ -13,38 +13,92 @@
 
 .carbon-dialog-full-screen__content {
   padding: 30px 5%;
-}
+  overflow: scroll;
+  height: calc(100% - 60px);
+
+  > .carbon-carousel {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    left: 0;
+    top: 0;
+
+    .carbon-carousel__content {
+      overflow: scroll;
+      position: initial;
+      height: 100%;
+    }
```

Comment:
Wouldn't you argue the other way around if this css lives in `carousel.scss` that it should live in `dialog-full-screen.scss` instead?